plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/LinuxExec.kt

```diff
@@ -128,10 +137,11 @@ open class LinuxExec @Inject constructor(objects: ObjectFactory) : ConclaveTask(
             errorOut.writeTo(System.err)
             // Using default charset here because the strings come from a sub-process and that's what they'll pick up.
             // Hopefully it's UTF-8 - it should be!
-            return String(errorOut.toByteArray()).split(System.lineSeparator())
+            val errorMessage =  String(errorOut.toByteArray())
+            throw GradleException(errorMessage)
         }
         result.assertNormalExitValue()
-        return null
+        return String(stdOut.toByteArray()).split(System.lineSeparator())
```

Comment:
We shouldn't really split the output here, it's the caller's job to do that if they wish. It's only bc of `LinuxExec` we do this. I think the splitting should go there, but it's not really important for this PR.